### DISCUSSION TITLE
Sylius and Accessibility

### DISCUSSION THREAD

**Discussion by Menelion** [Sylius/Sylius#16711] [Q&A] [answered]
Hi Team and Community,

I'm in search of an eCommerce solution, and I have two main requirements: written in PHP and accessible for various categories of users. We are a small team, and two of us including me are totally blind, and even among our sighted members there are accessibility specialists, so accessibility must be in the DNA of our solution.  
I looked at Sylius and got very enthusiastic because first, I know and love Symfony, and second, the solution (at least from the demo) seems not perfectly accessible but it's quite manageable, especially if the Sylius team accepts pull requests.

So my main question is: What is the relation of the team towards accessibility? Do you care about it, do you think about it, are you willing to receive help in form of both pull requests and hands-on testing? I know that it's difficult to answer "No!" to such questions, but still, for me honesty is far more prevalent than good appearance, if you know what I mean. I had very painful experience of tailoring an inaccessible ECommerce solution to a client's needs while struggling myself, and it was… the last thing I want to be repeated in my career.  
If you want to say something non-publicly, please feel free, my email address is public. Thank you.

**Comment by loic425** [comment, marked as answer]
Hi @Menelion 
The latest release is WCAG compliant, you can take a look on the release blogpost https://sylius.com/blog/news/first-minor-release-in-the-sylius-2-x-era/

**Comment by Rafikooo** [comment]
Closing this discussion as Sylius since version 2.1 is WCAG compliant.
